src/Chapter2.hs

```diff
@@ -1,3 +1,4 @@
+{- ORMOLU_DISABLE -}
```

Comment:
🙏🏻

Reply:
yea, otherwise the whole pull request will be full of whitespace changes LOL